src/Controls/tests/UITests/Tests/LabelUITests.cs

```diff
@@ -32,6 +32,11 @@ public void SpanTapped()
 			Assert.Ignore("Click (x, y) pointer type mouse is not implemented.");
 		}
 
+		if (Device == TestDevice.Windows)
+		{
+			Assert.Ignore("GetRect not implemented on Windows.");
```

Comment:
```
[WinAppDriver] Command not implemented: GET: /session/3B433044-76E3-4017-BEAC-4911755824F9/element/42.6554534.4.405/rect
[WD Proxy] Got response with status 501: Command not implemented: GET: /session/3B433044-76E3-4017-BEAC-4911755824F9/element/42.6554534.4.405/rect
[W3C (22c5ea62)] Encountered internal error running command: Error: Could not proxy. Proxy error: Request failed with status code 501
```

Ignore Windows tests using the GetRect method https://w3c.github.io/webdriver/#get-element-rect
Getting an not implemented exception. Was working in previous versions.